src/index.js

```diff
@@ -367,9 +367,16 @@ function applyClassName({ attributes }) {
 
 
 function extend(base, props) {
```

Comment:
Nothing else used rest params, so I assumed we were in a non transpile land.

Props is now "unused" but the function length of Object.assign is 2 anyway, so I kept it.